libtransmission/variant.cc

```diff
@@ -55,7 +55,7 @@ static void tr_variant_string_clear(struct tr_variant_string* str)
 {
     if (str->type == TR_STRING_TYPE_HEAP)
     {
-        delete[]((char*)(str->str.str));
+        delete[] ((char*)(str->str.str));
```

Comment:
This is a sign that you've been skipping the so-called "pre-commit" hook.
You need to use clang-format to validate your code formatting. Or, in short, you can't add a space here.